bobtask/artifact.go

```diff
@@ -73,7 +73,11 @@ func (t *Task) ArtifactPack(artifactName hash.In) (err error) {
 		exports = append(exports, filepath.Join(t.dir, path.String()))
 	}
 
+	// TODO: reestablish the local store.
 	artifact, err := t.local.NewArtifact(context.TODO(), artifactName.String())
+
+	// TODO: Currently the remote store ignores the username.
+	// artifact, err := t.remote.NewArtifact(context.TODO(), artifactName.String())
```

Comment:
I suppose this can be removed? (We sync at the end anyway)

Reply:
Yes pleas remove.